st2common/st2common/models/db/action.py

```diff
@@ -215,6 +209,9 @@ class ActionAliasDB(stormbase.StormBaseDB):
     formats = me.ListField(
         field=me.StringField(),
         help_text='Possible parameter formats that an alias supports.')
+    file_uri = me.StringField(
```

Comment:
ActionAlias does not have a pack yet. Not sure if it should etc. so left that out for now and not inheriting from `stormbase.StormContentDB`